docs/topics/logging.rst

```diff
@@ -242,6 +242,48 @@ e.g. in the spider's ``__init__`` method::
 If you run this spider again then INFO messages from
 ``scrapy.spidermiddlewares.httperror`` logger will be gone.
 
+You can also filter messages by content using a substring or
+a regular expression. Create a ``logging.Filter`` subclass 
```

Comment:
It would be best to link the first references to new API objects to their reference documentation, e.g.

```suggestion
a regular expression. Create a :class:`logging.Filter` subclass 
```